src/tests/calculate.test.js

```diff
@@ -0,0 +1,94 @@
+import calculate from '../logic/calulate';
+
+describe('Testing arithmetic operations', () => {
+  it('sum up', () => {
```

Comment:
Let's make sure our `it` block sentence is explanatory. `sum up` or `falsy sum` doesn't really explain the error if the test failed.